app/controllers/SingleRecordController.scala

```diff
@@ -42,10 +42,10 @@ class SingleRecordController @Inject() (
     goodsRecordConnector.getRecord(request.eori, recordId).map { record =>
       val detailsList = SummaryListViewModel(
         rows = Seq(
-          TraderReferenceSummary.row(record.traderRef, recordId, NormalMode),
-          GoodsDescriptionSummary.row(record.goodsDescription, recordId, NormalMode),
-          CountryOfOriginSummary.row(record.countryOfOrigin, recordId, NormalMode),
-          CommodityCodeSummary.row(record.commodityCode, recordId, NormalMode),
+          TraderReferenceSummary.row(record.traderRef, recordId, CheckMode),
```

Comment:
should be normal mode as it is the start of a journey - not a check your answers page

Reply:
Updated